src/StepFunctionsAutoDiscover.ts

```diff
@@ -97,9 +122,20 @@ export class StepFunctionsStateMachine extends Component {
     src.line('import fs from \'fs\';');
     src.line('import path from \'path\';');
 
+    let isYaml = /(yaml|yml)/.test(extension);
+    if (isYaml) {
+      src.line('import * as yaml \'js-yaml\';');
```

Comment:
here is missing 'from', so that it looks like:

import * as yaml from \'js-yaml\';'

Sorry, i've overseen it by review.